examples/utils_squad.py

```diff
@@ -908,9 +929,7 @@ def _strip_spaces(text):
     # and `pred_text`, and check if they are the same length. If they are
     # NOT the same length, the heuristic has failed. If they are the same
     # length, we assume the characters are one-to-one aligned.
-    tokenizer = BasicTokenizer(do_lower_case=do_lower_case)
-
-    tok_text = " ".join(tokenizer.tokenize(orig_text))
+    tok_text = " ".join(tokenizer.tokenize(orig_text)) if tokenizer else orig_text
```

Comment:
Here, we need to use `BasicTokenizer` instead of model-specific tokenizer which can split a token into multiple tokens, and inject wordpiece prefixes (`##`).

Reply:
But `BasicTokenizer` has its own vocabulary. Isn't a conflict with a model-specific tokenizer?

Reply:
If I understand correctly, `BasicTokenizer` does not have its own vocabulary. It is just a simple rule-based tokenizer with text normalization.

Reply:
Yes, you are right. There we need to keep `BasicTokenizer`.